README.md

```diff
@@ -1,60 +1,51 @@
 ## Shop This City
+
   A mobile friendly application build on iOS Platform using React Native. It allows Users to favourite items from assorted well-known brands by their categories which enables the User to build up their own Pinterest Board. A divine feature that allows a User to go back to the items easily when the item has not been purchased. 
   
   # SnapShots
  
- ### Browse Screen 
-
-![Simulator Screen Shot - iPhone X - 2019-09-13 at 13 43 06](https://user-images.githubusercontent.com/43800526/64897394-1deb8e00-d638-11e9-912a-2db10d0bc12d.png)
-
-
-  ### Store Screen  
  
-![Simulator Screen Shot - iPhone X - 2019-09-13 at 13 43 31](https://user-images.githubusercontent.com/43800526/64897369-06140a00-d638-11e9-8e65-22e07a1d391b.png)
-
+![Alt Text] (https://media.giphy.com/media/YPtCW7ow8HpTiisAPP/giphy.gif)
 
-### Brand Screen 
 
-![Simulator Screen Shot - iPhone X - 2019-09-13 at 13 43 34](https://user-images.githubusercontent.com/43800526/64897341-f0064980-d637-11e9-88eb-cbdbe8dfa014.png)
 
+# Instructions to run the project 
 
-### New Arrivals 
+```Git Clone the repository``` 
 
-![Simulator Screen Shot - iPhone X - 2019-09-13 at 13 43 46](https://user-images.githubusercontent.com/43800526/64897309-d1a04e00-d637-11e9-8e3f-e0faa4190614.png)
+```cd into client dir```
 
+```yarn install``` 
 
-### Sale 
+```cd into ios``` 
 
-![Simulator Screen Shot - iPhone X - 2019-09-13 at 13 43 50](https://user-images.githubusercontent.com/43800526/64897252-9ef65580-d637-11e9-8594-9b9c06664449.png)
+```pod install```
 
+```go back to client dir```
 
+run your app  with  ```react-native run-ios ```
 
 
-# Instructions to run the project 
+# Technologies: 
```

Comment:
It should not be an H1 title. Use the same level of heading as other sections in your README.